exporter/schema.py

```diff
@@ -35,3 +38,17 @@ def latest_links_schema(self) -> SchemaResource:
         )[0]
 
         return SchemaResource.from_dict(latest_schema)
+
+    @lru_cache()
```

Comment:
when does this clear? does it mean we have to redeploy the exporter when we upgrade file descriptor, links schema?  which I think is fine because i don't think that will be updated often. We just need to keep note of it.

Reply:
Good point yeah. This will clear when the object itself (an instance of SchemaService()) goes out of scope, which will only happen when an exporter pod is redeployed, since `schema_service` is a singleton created in exporter.py. 

We could instead re-instantiate a SchemaService() for each time we receive an assay message. Also, could replace lru_cache with a timed cache, like https://cachetools.readthedocs.io/en/stable/#cachetools.TTLCache

Reply:
@aaclan-ebi I added the TTLCache from above, schemas will only be cached for a maximum of 5 minutes